js/services/vCardProperties.js

```diff
@@ -91,15 +91,18 @@ angular.module('contactsApp')
 		impp: {
 			multiple: true,
 			readableName: t('contacts', 'Instant messaging'),
-			template: 'text',
+			template: 'username',
 			defaultValue: {
 				value:[''],
-				meta:{type:['HOME']}
+				meta:{type:['SKYPE']}
 			},
 			options: [
-				{id: 'HOME', name: t('contacts', 'Home')},
-				{id: 'WORK', name: t('contacts', 'Work')},
-				{id: 'OTHER', name: t('contacts', 'Other')}
+				{id: 'IRC', name: t('contacts', 'IRC')},
```

Comment:
Could you change `t('contacts', 'IRC')` to `'IRC'` only - the translation is not really needed for those strings.